haystack/file_converter/base.py

```diff
@@ -101,9 +108,27 @@ class FileTypeClassifier(BaseComponent):
     """
     outgoing_edges = 5
 
-    def run(self, file_path: Path, **kwargs):  # type: ignore
-        output = {"file_path": file_path, **kwargs}
-        ext = file_path.name.split(".")[-1].lower()
```

Comment:
`Path.suffix` basically does this operation for you!

Reply:
Yup. I will refactor the code.